### DISCUSSION TITLE
Information on Polarfire SoC Application Core control and reset

### DISCUSSION THREAD

**Discussion by ManishaI31214** [polarfire-soc/polarfire-soc-documentation#127] [Reference designs, FPGA and development kits] [answered]
How to control each CPU core dynamics individually so that in case of a problem with one core, it can restart again without affecting other cores?
We know that PMP mode can separate each peripheral in each context. But if context A is crashed. Is there any possible way to recover the context A core without affecting context B cores

**Comment by hughbreslin** [comment, marked as answer]
Hi @ManishaI31214 

This falls under the reboot command work we're currently doing. Its planned for our next release in 2022.09. We'll have a solution for restarting the individual contexts / harts then 🙂

**Comment by ManishaI31214** [comment]
Thank you
